src/index.js

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { BrowserRouter as Route, Router } from 'react-router-dom';
```

Comment:
here as you can see `as` is used while importing the module from `react-router-dom`. when you define alias like above, you should also follow the name you defined. so I just replaced `<BrowserRouter>` with `<Route>`

for the further research, I recommend to have a look at Modern JavaScript Tutorial's `import` chapter 👍